### DISCUSSION TITLE
Can we disable to S-h, S-l to default nvim map ?

### DISCUSSION THREAD

**Discussion by Cruising0904** [LazyVim/LazyVim#570] [Q&A] [answered]
I understand most of key settings are related to respective plugins though, I actually use a lot Shift + h and Shit+ l , instead of this I use alt key How can I remap those keys to be back to default nvim key?

**Comment by dpetka2001** [comment, marked as answer]
I assume you can just put in `lua/config/keymaps.lua` whatever change you want to do. Just take a look at the [docs](https://www.lazyvim.org/configuration/plugins#%EF%B8%8F-adding--disabling-plugin-keymaps) and make the appropriate changes for what you want. Currently those keymaps are binded to prev_buffer/next_buffer. So, just change them to what you like.

**Reply by Cruising0904** [reply]
Thanks for the tip!. I couldn't find the name of {rhs} , I just unmapped those thanks to you :)
